app/validators/existing_path_validator.rb

```diff
@@ -0,0 +1,5 @@
+class ExistingPathValidator < ActiveModel::EachValidator
```

Comment:
This is fun. I didn't know you could do that.

Reply:
Yeah, good innit!